### ISSUE TITLE
CLI changed packages should be sorted alphabetically

### ISSUE THREAD

**Issue by bengeorge** [changesets/changesets#1077]
## Affected Packages

CLI

## Problem

Ordering of "changed packages" is by **length**

eg:

```
changed packages
  @foo/dddd
  @foo/aaa
  @foo/bb
```

## Proposed solution

Sort packages by alphabetical order

```
changed packages
  @foo/aaa
  @foo/bb
  @foo/dddd
```

**Comment by Andarist** [comment]
Hm, I wonder if they shouldn't be sorted alphabetically but within "groups" - where a group would be created but the closest common ancestor directory. Perhaps I'm overthinking this but it kinda makes sense for those 2 patterns to create 2 groups like this:
```json
"workspaces": ["apps/*", "packages/*"] 
```

WDYT?

**Comment by Filipoliko** [comment]
Hi, we ran into the same issue with a repository with over 50 packages and it can be very confusing when searching for a specific package.

I created a PR #1122, which should resolve this issue by sorting packages alphabetically (but not by groups).

The groups sorting does not seem to be a problem for our use-case, since each group for us means different package scope so as long as the sorting works, the groups are also together.

**Comment by Andarist** [comment]
@Filipoliko would groups-based sorting go against your needs?

**Comment by Filipoliko** [comment]
@Andarist No, for our use-case it doesn’t really matter if the groups are there or not.

**Comment by corvidism** [comment]
@Andarist @Filipoliko was there any progress on this? The idea about groups is neat, but even simple alphabetical order would be an improvement. It really is a pain when there's more than a handful of packages in a repo.

**Comment by jmatsushita** [comment]
I would also be happy with only the alphabetical order improvement. I'm using this with a patch and it's so much better! 

https://github.com/changesets/changesets/blob/ae3fd61fbcc8e6a508adc773f6937056fda50b5e/packages/cli/src/commands/add/createChangeset.ts#L69-L76

**Comment by bluwy** [comment]
Now sorted in the next v3 major.
